applications/openshift/authentication/oauthclient_inactivity_timeout/rule.yml

```diff
@@ -65,4 +65,4 @@ template:
         yamlpath: ".items[:].accessTokenInactivityTimeoutSeconds"
         check_existence: "all_exist"
         entity_check: "all"
-        xccdf_variable: var_oauth_inactivity_timeout
\ No newline at end of file
+        xccdf_variable: var_access_token_inactivity_timeout_seconds
```

Comment:
Why do we change the variable here as well? The variable is used in multiple rules:
```
applications/openshift/authentication/oauth_inactivity_timeout/rule.yml:         accessTokenInactivityTimeout: {{{ xccdf_value("var_oauth_inactivity_timeout") }}}
applications/openshift/authentication/oauth_inactivity_timeout/rule.yml:  the output should return <pre> {{{ xccdf_value("var_oauth_inactivity_timeout") }}} </pre>.
applications/openshift/authentication/oauth_inactivity_timeout/rule.yml:        xccdf_variable: var_oauth_inactivity_timeout
applications/openshift/authentication/oauth_or_oauthclient_inactivity_timeout/kubernetes/shared.yml:    accessTokenInactivityTimeout: {{.var_oauth_inactivity_timeout}}
applications/openshift/authentication/oauthclient_inactivity_timeout/rule.yml:        xccdf_variable: var_oauth_inactivity_timeout
controls/nist_ocp4.yml:  - var_oauth_inactivity_timeout=10m0s
controls/nist_ocp4.yml:  - var_oauth_inactivity_timeout=10m0s
~

```

Reply:
We need to have two variables `var_oauth_inactivity_timeout` and `var_access_token_inactivity_timeout_seconds` because we have two rules, one is checking cluster global timeout units in $m$s, and we have another rule that is checking oauth client timeout that has units in seconds

Reply:
rule `oauthclient_inactivity_timeout` and `oauth_inactivity_timeout`

Reply:
ah! So the two rules can nominally have the same timeout in seconds, but one has to be express really in seconds and the other can be expressed in human-readable units. Got it.